qubes/vm/qubesvm.py

```diff
@@ -801,6 +802,17 @@ def xid(self):
                 e.get_error_code()))
             raise
 
+    @qubes.stateless_property
+    def stubdom_uuid(self):
+        stubdom_xid = self.stubdom_xid
```

Comment:
pylint needs some "help": `# pylint: disable=comparison-with-callable`

Reply:
Actually, I have a better idea: https://github.com/QubesOS/qubes-core-admin/pull/602/commits/3362ffc7ed4ea33c479f1df24fbdb25417a3584a
So, don't disable the message (it will be okay once I merge the other PR).